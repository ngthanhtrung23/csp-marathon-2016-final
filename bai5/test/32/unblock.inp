1.2234
155534
00678.
99678.
...ABB
...ACC
