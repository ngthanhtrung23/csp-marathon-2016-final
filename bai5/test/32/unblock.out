37
2 L 1
6 D 2
0 R 1
9 R 1
1 D 4
0 L 1
5 L 1
7 U 2
9 L 1
6 U 2
A U 2
B L 3
4 D 3
C L 3
8 D 2
3 D 2
A D 2
7 D 2
2 R 2
5 R 3
6 U 2
0 R 1
9 R 1
1 U 4
0 L 1
9 L 1
B L 1
C L 1
6 D 4
0 R 1
2 L 2
9 R 1
1 D 2
5 L 3
3 U 2
7 U 2
0 R 3

